pep-0589.rst

```diff
@@ -0,0 +1,648 @@
+PEP: 589
+Title: TypedDict: Type Hints for Dictionaries with a Fixed Set of Keys
+Author: Jukka Lehtosalo <jukka.lehtosalo@iki.fi>
+Sponsor: Guido van Rossum <guido@python.org>
+Discussions-To: typing-sig@python.org
```

Comment:
It looks like this has not been posted to typing-sig yet. I think this looks very solid and is ready to be posted there.